extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/VarListChunkInputStreamGenerator.java

```diff
@@ -21,22 +21,22 @@
 import io.deephaven.util.mutable.MutableInt;
 import org.jetbrains.annotations.Nullable;
 
-import java.io.DataInput;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.util.Iterator;
-import java.util.PrimitiveIterator;
 
 public class VarListChunkInputStreamGenerator<T> extends BaseChunkInputStreamGenerator<ObjectChunk<T, Values>> {
     private static final String DEBUG_NAME = "VarListChunkInputStreamGenerator";
 
+    private final Factory factory;
```

Comment:
Future follow-up:
I'd sort of like to find a pattern where this isn't retained into a field, but just used in the constructor, since in theory we only need to _create_ the input stream generator - like creating `innerGenerator` etc up front, but not sure about that.

Also might want to move the factory instance to the last param rather than the first, like the ChunkReader apis.

Reply:
Maybe we can save a functor to go from innerChunk -> innerGenerator instead?